frontends/web/src/locales/en/app.json

```diff
@@ -134,7 +134,7 @@
   "bitbox02Settings": {
     "deviceName": {
       "input": "BitBox02 Name",
-      "title": "Set BitBox 02 name"
+      "title": "Set BitBox02 name"
```

Comment:
Changing values needs to be done inside locize, see https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/docs/i18n.md

I went in there and fixed this one. You can remove this change from this PR.